pkg/preflight/aws.go

```diff
@@ -0,0 +1,133 @@
+package preflight
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/opsworks"
+	"github.com/sirupsen/logrus"
+
+	"github.com/libopenstorage/cloudops"
+	awsops "github.com/libopenstorage/cloudops/aws"
+)
+
+const (
+	eksDistribution = "eks"
+)
+
+type aws struct {
+	checker
+	ops  cloudops.Ops
+	zone string
+}
+
+func (a *aws) initCloudOps() error {
+	if a.ops != nil {
+		return nil
+	}
+
+	// Initialize aws client for cloud drive permission checks
+	// TODO: use provided credentials to create a client
+	client, err := awsops.NewClient()
+	if err != nil {
+		logrus.Infof("JLIAO: err: %v", err)
```

Comment:
Same for this file, remove all "JLIAO" lines or make them into more permanent ones